contrib/exportLUT.lua

```diff
@@ -0,0 +1,139 @@
+--[[
+    This file is part of darktable,
+    copyright (c) 2020 Noah Clarke
+
+    darktable is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    darktable is distributed in the hope that it will be useful,
+     but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with darktable.  If not, see <http://www.gnu.org/licenses/>.
+]]
+--[[
+Add the following line to .config/darktable/luarc to enable this lightable module: 
+  require "contrib/exportLut"
+
+Given a haldCLUT identity file this script generates haldCLUTS from all the user's
+styles and exports them to a location of their choosing.
+
+Warning: during export if a naming collision occurs the older file is automatically
+overwritten silently.
+]]
+
+local dt = require "darktable"
+local du = require "lib/dtutils"
+local df = require("lib/dtutils.file")
+local ds = require("lib/dtutils.system")
+
+local gettext = dt.gettext
+
```

Comment:
Add this
  ```
gettext.bindtextdomain("exportLUT",dt.configuration.config_dir.."/lua/locale/")

  local function _(msgid)
      return gettext.dgettext("exportLUT", msgid)
  end
```
then you can replace your gettext.gettext with "_", which matches the convention used in the darktable source as well as the other scripts.

Reply:
Should be fixed now, thanks for the heads up!